backend/.env.example

```
@@ -47,12 +47,15 @@ REDIS_PORT=6379
 
 # RustFS is an S3-compatible object storage (uses boto3)
-# External endpoint: https://s3.evillab.dev
-RUSTFS_HOSTNAME=s3.evillab.dev
+# Internal API endpoint (boto3 client connects here)
+RUSTFS_HOSTNAME=s3-api.evillab.dev
 RUSTFS_PORT=443
-RUSTFS_ACCESS_KEY=your-access-key
-RUSTFS_SECRET_KEY=your-secret-key
+RUSTFS_USE_HTTPS=True
+# Shared dev credentials — do NOT use in production
+RUSTFS_ACCESS_KEY=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RUSTFS_ACCESS_KEY key should go before the RUSTFS_HOSTNAME key

(UnorderedKey)